ros_packages/motors/pib_motors/pib_motors/bricklet_pin.py

```diff
@@ -39,21 +44,26 @@ def get_settings(self) -> dict[str, Any]:
 		except Exception as error: logging.error(f'error occured while trying to get motor-settings: {str(error)}')
 		return settings_dto
 	
+	def get_current(self) -> int:
+		"""returns the current of the bricklet pin, or NO_CURRENT, if it is not connected"""
+		return self.bricklet.get_servo_current() if self.is_connected() else BrickletPin.NO_CURRENT
+	
 	def is_connected(self) -> bool:
+		"""checks if the bricklet-pin is connected to a bricklet"""
 		# X,Y and Z are the default uid of a Servo Bricklet 2.0 (updated ob boot by update_bricklet_uids.py)
-		if self.bricklet.uid_string != 'X' and self.bricklet.uid_string != 'Y' and self.bricklet.uid_string != 'Z':
-				return True
-		return False
+		return self.bricklet.uid_string != 'X' and self.bricklet.uid_string != 'Y' and self.bricklet.uid_string != 'Z'
```

Comment:
The default uids has been changed to "AAA", "BBB", "CCC"

Reply:
changed